docs_overview.md

```diff
@@ -0,0 +1,73 @@
+The MongoDB Stitch JavaScript SDK can be used to build web applications, Node.js applications, and React Native applications that use MongoDB Stitch.
+
+### Links to Other Resources
+
+[![Join the chat at https://gitter.im/mongodb/stitch](https://badges.gitter.im/mongodb/stitch.svg)](https://gitter.im/mongodb/stitch?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+[MongoDB Stitch Users - Google Group](https://groups.google.com/d/forum/mongodb-stitch-users)
+
+[MongoDB Stitch Announcements - Google Group](https://groups.google.com/d/forum/mongodb-stitch-announce)
+
+[MongoDB Stitch Documentation](https://docs.mongodb.com/stitch/)
+
+[JavaScript SDK GitHub Repository](https://github.com/mongodb/stitch-js-sdk)
+
+### Basic Usage
+
+#### Construct a simple app-wide client
```

Comment:
It'd be nice if there was a way to source this from the README or some single location in the event we need to make updates.

Reply:
It seems like we wouldn't be able to do this and have both our Github README and Overview have the format we want: https://github.com/github/markup/issues/172#issuecomment-33241601